src/passes/CMakeLists.txt

```diff
@@ -46,3 +46,9 @@ SET(passes_SOURCES
   Vacuum.cpp
 )
 ADD_LIBRARY(passes STATIC ${passes_SOURCES})
+IF(BUILD_STATIC_LIB)
+  SET_TARGET_PROPERTIES(passes PROPERTIES OUTPUT_NAME binaryen-passes)
```

Comment:
So the other targets would have a similar install rule that has everything other than this line?

Reply:
Unfortunately, yes, because of the shared - static lib build asymmetry in this project.

Reply:
Do you mean the fact that in shared mode we have one big dynamic library, whereas here we have one static library per module?
Actually come to think of it, for this use case (I'm assuming your use case is to install static libraries in a system to link into other programs), would it make more sense to also just have one big  one, e.g. `libbinaryen.a`? That seems more convenient to a user who doesn't care about the implementation details of Binaryen.

Reply:
Having single static library would be a bit more convenient, but that's required. I can handle multiple static libs in CMake exported config files.

Reply:
The bigger problem is that there are not public headers for these libraries, so we cannot use them.